opencog/guile/LoggerSCM.cc

```diff
@@ -140,29 +170,42 @@ LoggerSCM::LoggerSCM() : ModuleWrap("opencog logger") {}
 /// Thus, all the definitions below happen in that module.
 void LoggerSCM::init(void)
 {
-	define_scheme_primitive("cog-logger-set-level!",
+	define_scheme_primitive("cog-default-logger",
+		&LoggerSCM::do_default_logger, this, "logger");
+	define_scheme_primitive("cog-ure-logger",
+		&LoggerSCM::do_ure_logger, this, "logger");
```

Comment:
this doesn't make sense to me.  Why is this in C++? Why not just create something like this:
```
(define my-fav-logger (create-new-logger "/path/to/log/file"))
(cog-logger-set-loglevel my-fav-logger  "fine")
```
 and if you really want to get whizzy, stick a bunch of methods on it:
```
(define my-fav-logger (create-new-logger "/path/to/log/file"))
(my-fav-logger 'set-loglevel "fine")
```
FWIW, I'm very tempted to make the entire scheme API to opencog be object-oriented in this way, viz
```
(define my-atom (ConceptNode "foo"))
(my-atom 'set-tv (stv 0.42 0.69))
```
I don't know why I didn't do this from the very beginning.  What was I thinking?